.github/workflows/run-tests.yaml

```diff
@@ -3,10 +3,6 @@ run-name: run unit tests
 on:
   pull_request:
     branches-ignore: ["release-please*"]
-    paths:
```

Comment:
why do we have this? it interferes with some stuff (like if someone changed the setup.py build script or docs)

Reply:
I figured that we don't want to run unit tests if someone only changes some documentation etc. We can add more paths here, or switch to paths-ignore